src/main/java/hlf/java/rest/client/service/impl/NetworkStatusImpl.java

```diff
@@ -95,62 +106,85 @@ public ResponseEntity<ClientResponseModel> generateConfigUpdate(
       String channelName, NewOrgParamsDTO organizationDetails) {
     Network network = gateway.getNetwork(channelName);
 
+    if (network != null) {
+      ConfigUpdate.Builder configUpdateBuilder = createConfigUpdate(channelName);
+      ConfigUpdate configUpdate =
+          configUpdateBuilder
+              .setWriteSet(
+                  addOrgToChannelWriteSetBuilder.buildWriteset(
+                      configUpdateBuilder.getReadSet(), organizationDetails))
+              .build();
+      MessageOrBuilder message = configUpdate;
```

Comment:
redundant code

Reply:
`configUpdate` is getting used in later lines of code to get the byte array. So I cannot remove the assigment part.

Reply:
I am talking about line 117 only, `message` isn't used

Reply:
Understood, handled while refractoring.